src/languages/StandardSqlFormatter.js

```diff
@@ -68,7 +71,11 @@ const reservedNewlineWords = [
     "OR", "OUTER APPLY", "OUTER JOIN",
     "RIGHT JOIN", "RIGHT OUTER JOIN",
     "WHEN",
-    "XOR"
+    "XOR",
+    "PARTIONED BY",
```

Comment:
These are rather ad-hoc and based off of the grammar provided in the databricks docs. These keywords are used in various other places within SparkSQL, but I think that starting a new line for these should be fine